### ISSUE TITLE
PointPillars' training logs confusion 

### ISSUE THREAD

**Issue by TuanTNG** [open-mmlab/mmdetection3d#1463]
Hi MMDetection3D teams,

I am re-producing PointPillars config, [this config](https://github.com/open-mmlab/mmdetection3d/blob/master/configs/pointpillars/hv_pointpillars_secfpn_6x8_160e_kitti-3d-3class.py). However, when I read its training logs, [this logs](https://download.openmmlab.com/mmdetection3d/v1.0.0_models/pointpillars/hv_pointpillars_secfpn_6x8_160e_kitti-3d-3class/hv_pointpillars_secfpn_6x8_160e_kitti-3d-3class_20220301_150306.log.json),  I found something wrong. 

The total number of training samples of KITTI is 3712 as in [this readme](https://github.com/open-mmlab/mmdetection3d/blob/master/configs/3dssd/README.md). When training PointPillars [configs](https://github.com/open-mmlab/mmdetection3d/blob/master/configs/pointpillars/hv_pointpillars_secfpn_6x8_160e_kitti-3d-3class.py) with repeating KITTI dataset by 2 times, samples_per_gpu=6 and 8 GPUs. It must result in around `155` (~=3712*2/8/6) iterations per one epoch. However, the [training log](https://download.openmmlab.com/mmdetection3d/v1.0.0_models/pointpillars/hv_pointpillars_secfpn_6x8_160e_kitti-3d-3class/hv_pointpillars_secfpn_6x8_160e_kitti-3d-3class_20220301_150306.log.json) shows `1200` iterations per one epoch.

Is there anything wrong? 

I found that [other config](https://github.com/open-mmlab/mmdetection3d/blob/master/configs/3dssd/3dssd_4x4_kitti-3d-car.py) and [its log](https://download.openmmlab.com/mmdetection3d/v1.0.0_models/3dssd/3dssd_4x4_kitti-3d-car/3dssd_4x4_kitti-3d-car_20210818_203828.log.json) which trained using 4 GPUs & samples_per_gpu=4,  are correct (the log shows 450 (~= 3712*2/4/4) iterations per epoch).

Thank you so much.

**Comment by Xiangxu-0103** [comment]
Yeah, I found the training log for pp_kitti-3class is largely different from the corresponding config.

**Comment by Xiangxu-0103** [comment]
The potential difference has been fixed in #1486. I found the training log for pp_3class is exactly trained with only one GPU, which causes the iteration `1200` per epoch.

**Comment by Tai-Wang** [comment]
@ZCMax Please have a look at this issue and update the log files if necessary.
